INTEROP.md

```diff
@@ -20,60 +20,51 @@ imported into an IDE for the purpose of code completion and navigation.
  Interoperability with Swift/Objective-C is provided too and covered in a
 separate document [OBJC_INTEROP.md](OBJC_INTEROP.md).
 
+## Platform libraries ##
+
+ Note that in many cases there's no need to use mechanisms described below, as for APIs available on the
```

Comment:
The entire document is "below", so this note is very confusing.

Reply:
Done.